src/domain/routes/index.types.d.ts

```diff
@@ -0,0 +1,30 @@
+type ProbeForRouteArgs = import("lightning").ProbeForRouteArgs
+type RawHopWithNumbers = NonNullable<ProbeForRouteArgs["routes"]>[number][number]
+
+type PayViaPaymentDetailsArgs = import("lightning").PayViaPaymentDetailsArgs
+type RawHopWithStrings = NonNullable<PayViaPaymentDetailsArgs["routes"]>[number][number]
+
+type PayViaRoutesArgs = import("lightning").PayViaRoutesArgs
+type RawRouteFromProbe = PayViaRoutesArgs["routes"][number]
+type RawRoute = { safe_fee: number } & RawRouteFromProbe
+type Route = { roundedUpFee: Satoshis; roundedDownFee: Satoshis }
+
+type CachedRoute = { pubkey: Pubkey; route: RawRoute }
+
+interface IRoutesRepository {
```

Comment:
now I have doubts with this service, although we use redis is part of `bitcoin/lightning` domain/service

Reply:
Hmm I'm not sure. It maybe comes closer to the lightning domain but I'm not sure it quite falls under the lnd `service`.

Are you thinking to have the caching method definitions under the `ILightningService` interface, or to simply move all the files from the `domain/routes` dir to be in the `domain/bitcoin/lightning` dir?